pages/_conferenceCode/contacts/index.vue

```diff
@@ -544,22 +638,14 @@ function getAttendanceValue (filter){
   return registrationStatus || undefined
 }
 
-async function getMeetingsList (){
-  const query = {
-    FullText: ''
-  }
-  const rows = await this.$kronosApi.getMeetings(query)
-
-  return rows
-}
-
 function clearSelectedOptions (type){
```

Comment:
`switch` are not nice to use. 

you can simplify the function buy passing the name of the filter or using inline if()

```suggestion
function clearSelectedOptions (field, value){
  this.filter[field] = value;
}
```

```suggestion
function clearSelectedOptions (type){

  if(type=='meetings')          this.filter.selectedMeetings         = []
  if(type=='organization') {    this.filter.selectedOrganization     = []
                                this.organizationOptions             = [] }
  if(type=='attendance')        this.filter.selectedAttendance       = []
  if(type=='flag')              this.filter.selectedFlag             = []
  if(type=='country'            this.filter.selectedCountry          = []
  if(type=='scope')             this.filter.selectedScop             = ''
  if(type=='organization-type') this.filter.selectedOrganizationType = ''
}
```
just make it easy to read

Reply:
Done